client/src/App.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import styled from 'styled-components';
+import Header from './components/Header';
+import NavBar from './components/NavBar';
+import './App.css';
+import data from './data/db';
+import FavoriteIconDeactivated from './icons/favorite_icon_deactivated.svg';
+
+function App() {
+  console.log(data);
+
+  return (
+    <Wrapper>
+      <Header />
+      <Main>
+      <StyledUList>
+        {data.map(station => (
+          <StyledGasStationItem className="ListItems" key={station.uuid}>
+            {station.e5}{' '}
+            {station.name.length > 14 ? `${station.name.substring(0, 14)}...` : station.name}{' '}
+            {station.street} {station.post_code} {station.city}
+            <img src={FavoriteIconDeactivated} alt="Favorite icon deactivated" />
+          </StyledGasStationItem>
+        ))}
+      </StyledUList>
+      </Main>
+      <NavBar />
+    </Wrapper>
+  );
+}
+
+const Wrapper = styled.div`
+  display: flex;
+  width: 100%;
+  justify-content: center;
+  
+`;
+
+const Main = styled.main`
+  display: flex;
+  width: 70vw;y<
```

Comment:
We have already noticed the bug and have fixed it.

Reply:
Then why didn't you push it to the branch? The error is still there.